evaluate/eval.py

```diff
@@ -159,17 +157,11 @@ def compute_precision(self):
     def compute_f1(self):
         prec = self.compute_precision()
         rec = self.compute_recall()
-        try:
-            f1 = 2*(prec*rec)/(prec+rec)
-        except ZeroDivisionError as e:
-            f1 = -1.0
+        f1 = 2*(prec*rec)/(prec+rec)
```

Comment:
Why did this change to avoid zero division?

Reply:
If `prec + rec` is `0`, then I think the repair process has done something _exceptionally_ wrong and we should explicitly raise this instead of silently setting `f1 = -1`.

Reply:
I agree with @richardwu . The -1 was to print something that will indicate that we are doing something bad. But at the same time I wanted to see the report on detected errors, corrected cells et. Can we actually add two print statements in the report: 1) a statement that returns the detailed counts in the current report and 2) a statement that returns f1, precision, recall etc.

Reply:
ok for now